getcov

```diff
@@ -112,7 +113,13 @@ exclude_data() {
 
   LCOV --remove "${LCOV_INFO}" "Developer/SDKs/*" -d "${OBJ_DIR}" -o "${LCOV_INFO}"
   LCOV --remove "${LCOV_INFO}" "main.m" -d "${OBJ_DIR}" -o "${LCOV_INFO}"
-  # Remove other patterns here...
+
+  #Remove anything the .xcodecoverageignore file has specified should be ignored.
+  (cat "${SRCROOT}/.xcodecoverageignore"; echo) | while read IGNORE_THIS; do
```

Comment:
Any interest in allowing for comments (lines starting with `#`?) in the ignore file?